setup.py

```diff
@@ -28,7 +28,8 @@ def find_version(*file_paths):
             'docutils>=0.10',
             'rsa>=3.1.2,<=3.5.0',
             's3transfer>=0.1.12,<0.2.0',
-            'PyYAML>=3.10,<=3.13']
+            'ruamel.yaml>=0.15.0,<0.16.0',
```

Comment:
Should we be using 0.15.0? There's a note about staying away from it for now in their docs: https://yaml.readthedocs.io/en/latest/basicuse.html. May be worth putting a TODO about upgrading here.

Reply:
Might be some docs out of date. On the [readme](https://pypi.org/project/ruamel.yaml/) it says:

> For production systems you should pin the version being used with ``ruamel.yaml<=0.15``

More to the point, we need the newer versions to support 3.7. Would also like to use some of the new interface. How about we pin to a specific version for now?